site/content/en/blog/releases/1.20.0-rc.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "1.20.0-Release Candidate"
+linkTitle: "1.20.0-rc"
+date: "2022-01-04"
+---
+This is the 1.20.0-rc release of Agones.
+
+This release brings us the simple game server version 0.6 and grpc client and server on same version
+
+Check the <a href="https://github.com/googleforgames/agones/tree/release-1.20.0-rc" data-proofer-ignore>README</a> for details on features, installation and usage.
+
+**Implemented enhancements:**
+
+Update the simple game server version to 0.6.[#2413](https://github.com/googleforgames/agones/pull/2413)
+Update the minikube driver flag name (--vm-driver is deprecated) [#2399](https://github.com/googleforgames/agones/pull/2399)
+Unreal SDK misses "const" prefix for function parameters. [#2411](https://github.com/googleforgames/agones/issues/2411)
+TLS handshake error with multi-cluster setup [#2402](https://github.com/googleforgames/agones/issues/2402)
+Upgrade-gRPC-version [#2414](https://github.com/googleforgames/agones/pull/2414)
+Update the simple game server to move itself back to the Ready state after allocation [#2409](https://github.com/googleforgames/agones/pull/2409)
```

Comment:
Sorry, I really should have looked at this closer, my apologies, but this is not following the process - this should be copy / pasted from the changelog.

```suggestion
**Implemented enhancements:**

- Update the simple game server to move itself back to the Ready state after allocation [\#2409](https://github.com/googleforgames/agones/pull/2409) ([roberthbailey](https://github.com/roberthbailey))
- Add a flag to simple-game-server to shutdown after a specified number of seconds [\#2407](https://github.com/googleforgames/agones/pull/2407) ([roberthbailey](https://github.com/roberthbailey))

```

If there are items we think should be marked as features, we should mark the issues as features during the issue/pr review when doing the release checklist.

This release document should match the CHANGELOG.

In the template we have:

```
{ insert enhancements from the changelog and/or security and breaking changes }
```

This should be a copy/paste operation from the generated markdown.